app/javascript/shared/components/ChatForm.vue

```diff
@@ -90,6 +91,20 @@ export default {
       }
       this.$emit('submit', this.formValues);
     },
+
+    fieldValidation(name, regex = '') {
```

Comment:
Please consider a better name for the method.

Reply:
Ok